docs/install/any-kubernetes-cluster.md

```diff
@@ -199,14 +199,17 @@ The following commands install Gloo and enable its Knative integration.
 
 The following commands install Istio and enable its Knative integration.
 
-<!-- TODO(https://github.com/knative/docs/issues/2166): Create streamlined instructions to inline -->
+1. Install a properly configured Istio
+
+   ```bash
+   kubectl apply --filename {{< artifact repo="net-istio" file="istio.yaml" >}}
+   ```
 
-1. [Installing Istio for Knative](./installing-istio.md)
```

Comment:
I'd probably still link to this as an "advanced guide", since this does not cover everything, but thanks for the simplification.